.github/V2_PROGRESS.md

```diff
@@ -40,8 +40,8 @@ x => done
 * [~] `ais-panel`
   * Just header & footer slot, doesn't get any child info
 * [x] `ais-powered-by`
-* [~] `ais-range-input` Alex
-* [~] `ais-range-slider` Alex
+* [x] `ais-range-input`
+* [-] `ais-range-slider` Alex
```

Comment:
this is by accident I guess?

Reply:
I thought that when this PR is merged it's gonna be done. So I went ahead and marked as done.